packages/analytics-react-native/android/src/main/java/com/amplitude/reactnative/AmplitudeReactNativeModule.kt

```diff
@@ -20,9 +20,10 @@ class AmplitudeReactNativeModule(private val reactContext: ReactApplicationConte
     }
 
     @ReactMethod
-    private fun getApplicationContext(shouldTrackAdid: Boolean, promise: Promise) {
+    private fun getApplicationContext(options: ReadableMap, promise: Promise) {
+        val trackAdid = if (options.hasKey("adid")) options.getBoolean("adid") else false
```

Comment:
Should adid be included in the TrackingOptions? Like how other mobile sdk did? 
https://github.com/amplitude/Amplitude-Kotlin/blob/main/android/src/main/java/com/amplitude/android/plugins/AndroidContextPlugin.kt#L120

Reply:
@yuhao900914
https://github.com/amplitude/Amplitude-TypeScript/blob/main/packages/analytics-types/src/config/react-native.ts#L28 ?